source/sass/patterns/atoms/image.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../variables/baselinegrid.scss";
+@import "../../variables/color.scss";
+@import "../../variables/image";
+
+.image {
+  @include block-spacing($start: $baselinegrid-space-medium, $end: $baselinegrid-space-medium);
+  @include border();
+  @include padding($image-inline-spacing, inline);
+  @include padding($image-inline-spacing, block);
```

Comment:
Value for block-axis padding should use the `$baselinegrid-space-extra_small` variable.